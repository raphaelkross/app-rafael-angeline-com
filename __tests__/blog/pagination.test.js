/* eslint-env jest */

import { shallow } from "enzyme";
import Pagination from "../../components/blog/pagination";

describe("Pagination", () => {
	it("should not render when there's no pages", () => {
		const props = {
			current: 0,
			pages: 0
		};

		const component = shallow(<Pagination {...props} />);

		expect(component.find("nav").exists()).toBeFalsy();
	});
});
