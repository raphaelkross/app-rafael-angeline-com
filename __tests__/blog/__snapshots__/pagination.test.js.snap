// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination should not render when there isn't pages 1`] = `null`;

exports[`Pagination should render complete layout 1`] = `
<nav
  className="jsx-3604835618"
>
  <div
    className="jsx-3604835618 page-numbers"
  >
    <a
      className="jsx-3604835618"
      href="/blog/1"
      onClick={[Function]}
    >
      1
    </a>
    <span
      className="jsx-3604835618 "
    >
      ...
    </span>
    <a
      className="jsx-3604835618"
      href="/blog/3"
      onClick={[Function]}
    >
      3
    </a>
    <a
      className="jsx-3604835618"
      href="/blog/4"
      onClick={[Function]}
    >
      4
    </a>
    <span
      className="jsx-3604835618 current"
    >
      5
    </span>
    <a
      className="jsx-3604835618"
      href="/blog/6"
      onClick={[Function]}
    >
      6
    </a>
    <a
      className="jsx-3604835618"
      href="/blog/7"
      onClick={[Function]}
    >
      7
    </a>
    <span
      className="jsx-3604835618 "
    >
      ...
    </span>
    <a
      className="jsx-3604835618"
      href="/blog/10"
      onClick={[Function]}
    >
      10
    </a>
  </div>
  <div
    className="jsx-3604835618 page-arrows"
  >
    <a
      className="jsx-3604835618 page-arrows-left"
      href="/blog/4"
      onClick={[Function]}
    >
      <svg
        className="jsx-3604835618 feather feather-chevron-left"
        fill="none"
        height="24"
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="1"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <polyline
          className="jsx-3604835618"
          points="15 18 9 12 15 6"
        />
      </svg>
    </a>
    <a
      className="jsx-3604835618 page-arrows-right"
      href="/blog/6"
      onClick={[Function]}
    >
      <svg
        className="jsx-3604835618 feather feather-chevron-right"
        fill="none"
        height="24"
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="1"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <polyline
          className="jsx-3604835618"
          points="9 18 15 12 9 6"
        />
      </svg>
    </a>
  </div>
</nav>
`;
